molecule/default/converge.yml

```diff
@@ -0,0 +1,65 @@
+---
+- name: Converge
+  hosts: all
+  become: true
+
+  vars:
```

Comment:
This attempts to make the environment and options similar to what is run by vagrant.